src/SFA.DAS.EmployerFinance.Web/Views/EmployerAccountTransactions/Index.cshtml

```diff
@@ -14,101 +14,159 @@
-            @if (Model.Data.ExpiringFunds.HasValue && Model.Data.ExpiringFunds > 0 && Model.Data.ExpiryDate.HasValue)
+
+<h2 class="heading-medium">Current</h2>
+
+<div class="grid-row current-funds">
+    <div class="column-one-third">
+        @{
+            var balance = Model.Data.CurrentLevyFunds.ToString("C0", culture);
+            var balanceClass = string.Empty;
+
+            if (balance.Length > 8)
             {
-                <div class="data data--long data--expired">
-                    <dl class="transfer-funds">
-                        @{
-                            var expiryDate = Model.Data.ExpiryDate.Value.ToString("MMMM yyyy");
-                        }
-                        <dt>
-                            of your funds could expire in <strong>@expiryDate </strong>
-                        </dt>
-                        @{
-                            var expiringFunds = Model.Data.ExpiringFunds.Value.ToString("C0", culture);
-
-                            var expiringFundsClass = string.Empty;
-
-                            if (balance.Length > 8)
-                            {
-                                expiringFundsClass = "long";
-                            }
-                            if (balance.Length > 10)
-                            {
-                                expiringFundsClass = "longer";
-                            }
-                        }
-                        <dd class="@expiringFundsClass">@expiringFunds</dd>
-                    </dl>
-                </div>
+                balanceClass = "long";
             }
-        </div>
+            if (balance.Length > 10)
+            {
+                balanceClass = "longer";
+            }
+        }
+        <div class="finance-figure @balanceClass" aria-labelledby="lbl-current-funds">@balance</div>
+        <p class="finance-figure__label" id="lbl-current-funds">Current funds</p>
+    </div>
+    <div class="column-one-third">
+        @{
+            var currentSpend = Model.Data.TotalSpendForLastYear.ToString("C0", culture);
+            var currentSpendClass = string.Empty;
+
+            if (currentSpend.Length > 8)
+            {
+                currentSpendClass = "long";
+            }
+            if (currentSpend.Length > 10)
+            {
+                currentSpendClass = "longer";
+            }
+        }
+        <div class="finance-figure @currentSpendClass" aria-labelledby="lbl-current-spent-funds">@currentSpend</div>
+        <p class="finance-figure__label" id="lbl-current-spent-funds">Funds spent in the last 12 months</p>
     </div>
+     @if (Model.Data.ExpiringFunds.HasValue && Model.Data.ExpiringFunds > 0 && Model.Data.ExpiryDate.HasValue)
+     {
+    <div class="column-one-third">
+        @{
+            var expiringFunds = Model.Data.ExpiringFunds.Value.ToString("C0", culture);
+
+            var expiringFundsClass = string.Empty;
+
+            if (expiringFunds.Length > 8)
```

Comment:
We could introduce a little helper function for deciding between the long and longer class names.